src/Rings/MPolyMap/MPolyQuo.jl

```diff
@@ -137,6 +137,7 @@ end
 
 # The two main evaluation methods
 function (F::MPolyAnyMap{<: MPolyQuoRing})(g)
+  @req parent(g) === domain(F) "g not in domain"
```

Comment:
@simonbrandhorst: Did you do this? This is wrong!

Reply:
Yet I did it. I thought I removed it. Maybe it comes from an intermediate commit?

Reply:
Who knows. But then at least we agree that this can be removed.